Library/Notifications.swift

```diff
@@ -18,19 +18,19 @@ public enum UserInfoKeys {
 }
 
 extension Notification.Name {
-  public static let ksr_sessionStarted = Notification.Name(rawValue: CurrentUserNotifications.sessionStarted)
-  public static let ksr_sessionEnded = Notification.Name(rawValue: CurrentUserNotifications.sessionEnded)
-  public static let ksr_userUpdated = Notification.Name(rawValue: CurrentUserNotifications.userUpdated)
-  public static let ksr_projectSaved = Notification.Name(rawValue: CurrentUserNotifications.projectSaved)
   public static let ksr_dataRequested = Notification.Name(rawValue: CurrentUserNotifications.dataRequested)
+  public static let ksr_environmentChanged = Notification.Name(rawValue:
+    CurrentUserNotifications.environmentChanged
+  )
+  public static let ksr_projectSaved = Notification.Name(rawValue: CurrentUserNotifications.projectSaved)
   public static let ksr_savedProjectEmptyStateTapped =
     Notification.Name(rawValue: CurrentUserNotifications.savedProjectEmptyStateTapped)
+  public static let ksr_sessionStarted = Notification.Name(rawValue: CurrentUserNotifications.sessionStarted)
+  public static let ksr_sessionEnded = Notification.Name(rawValue: CurrentUserNotifications.sessionEnded)
   public static let ksr_showNotificationsDialog =
     Notification.Name(rawValue: CurrentUserNotifications.showNotificationsDialog)
-  public static let ksr_languageChanged = Notification.Name(rawValue:
-    CurrentUserNotifications.languageChanged
-  )
-  public static let ksr_environmentChanged = Notification.Name(rawValue:
-    CurrentUserNotifications.environmentChanged
+  public static let ksr_userLocalePreferencesChanged = Notification.Name(rawValue:
```

Comment:
Same here, `ksr_userLocalePreferencesChanged` is the only new notification. The others are now alphabetized.